packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -289,7 +289,13 @@
             border-radius: 0 $BORDER_RADIUS $BORDER_RADIUS 0;
             border-top: 1px solid var(--general-section-1);
             background-color: var(--general-main-1);
-
+            @include tablet-screen {
+                max-width: 60rem;
+                margin: 0 auto;
+                padding: 1.6rem 0;
+                left: 0;
+                right: 0;
```

Comment:
lets make sure this works for rtl layout as well?
  inset-inline-start:0
    inset-inline-end:0

Reply:
I have removed this. 
as in new design we still have full width border to top of button